packages/react-native-windows-init/src/Cli.ts

```diff
@@ -0,0 +1,291 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import * as yargs from 'yargs';
+import * as fs from 'fs';
+import * as semver from 'semver';
+import {execSync} from 'child_process';
+import * as validUrl from 'valid-url';
+import * as prompts from 'prompts';
+import * as findUp from 'find-up';
+import * as chalk from 'chalk';
+
+const Registry = require('npm-registry');
+
+let npmConfReg = execSync('npm config get registry')
+  .toString()
+  .trim();
+let NPM_REGISTRY_URL = validUrl.isUri(npmConfReg)
+  ? npmConfReg
+  : 'http://registry.npmjs.org';
+const npm = new Registry({registry: NPM_REGISTRY_URL});
+
+const argv = yargs.options({
+  version: {
+    type: 'string',
+    describe: 'The version of react-native-windows to use.',
+  },
+  namespace: {type: 'string', describe: 'The native project namespace.'},
+  verbose: {type: 'boolean', describe: 'Enables logging.'},
+  language: {
+    type: 'string',
+    describe: 'Which language the app is written in.',
+    choices: ['cs', 'cpp'],
+    default: 'cpp',
+  },
+  overwrite: {
+    type: 'boolean',
+    describe: 'Overwrite any existing files without prompting',
+  },
+}).argv;
+
+const EXITCODE_NO_MATCHING_RNW = 2;
+const EXITCODE_UNSUPPORTED_VERION_RN = 3;
+const EXITCODE_USER_CANCEL = 4;
+
+function REACT_NATIVE_WINDOWS_GENERATE_PATH() {
```

Comment:
nit: I know this was like this before, but know why this function is cased like a constant? Seems off to me.